oxidation/parsec_cli/parsec_sequestre_dump/src/model.rs

```diff
@@ -0,0 +1,519 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use std::error::Error;
+
+use diesel::{ExpressionMethods, QueryDsl, RunQueryDsl, SqliteConnection};
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+use crate::schema::{block, info, vlob_atom};
+use parsec_api_crypto::SecretKey;
+use parsec_api_types::{DeviceID, FileManifest, FolderManifest, Manifest, WorkspaceManifest};
+
+type Time = chrono::DateTime<chrono::Utc>;
+
+#[cfg(test)]
+use crate::schema::{device, realm_role, user_ as user};
+#[cfg(test)]
+use tests_fixtures::Device;
+
+#[derive(Debug, Queryable, Serialize, Deserialize, PartialEq)]
```

Comment:
Why serialize/deserialize ? those are supposed to represent data from the sqlite database and are never going to be serialized as-is

Reply:
Done :+1: